web/web.go

```diff
@@ -556,23 +558,63 @@ func (h *Handler) graph(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handler) status(w http.ResponseWriter, r *http.Request) {
-	h.executeTemplate(w, "status.html", struct {
-		Birth          time.Time
-		CWD            string
-		Version        *PrometheusVersion
-		Alertmanagers  []*url.URL
-		GoroutineCount int
-		GOMAXPROCS     int
-		GOGC           string
+	status := struct {
+		Birth               time.Time
+		CWD                 string
+		Version             *PrometheusVersion
+		Alertmanagers       []*url.URL
+		GoroutineCount      int
+		GOMAXPROCS          int
+		GOGC                string
+		CorruptionCount     int64
+		ChunkCount          int64
+		TimeSeriesCount     int64
+		LastConfigTime      time.Time
+		ReloadConfigSuccess bool
 	}{
-		Birth:          h.birth,
+		// Match resolution of configuration reload time to avoid case where birth is after reload
```

Comment:
Why are you doing this?

Reply:
metric for reloading configuration is using `SetToCurrentTime` which is rounded to whole seconds. 
To avoid confusion I Truncate birth same way, so there is never case when reload time < birth time

Reply:
That doesn't sound right, the code shows it has ns resolution.

Reply:
https://github.com/prometheus/client_golang/blob/master/prometheus/gauge.go#L51
https://github.com/prometheus/client_golang/blob/master/prometheus/gauge.go#L99

Reply:
That has ns resolution. It's converted to a float before the division, and if it wasn't we'd fix it to be that way.